bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/siteadmin/aem61/SiteadminPage.java

```diff
@@ -0,0 +1,203 @@
+    toolbar.publishPageLater(localDateTime);
+    return this;
+  }
+
+  @Override
+  public SiteadminActions copyPage(String title, String destination) {
+    childPageWindow.selectPage(title);
+    toolbar.getCopyPageBtn().click();
+    driver.get(url + "/sites.html" + destination);
+    toolbar.getPastePageBtn().click();
+    return this;
+  }
+
+  @Override
+  public SiteadminActions createNewPage(String title, String name, String templateName) {
+    toolbar.getCreatePageBtn().click();
+
+    toolbar.getCreatePageActions()
+        .selectTemplate(templateName)
+        .provideName(name)
+        .provideTitle(title)
+        .submit();
+
+    return this;
+  }
+
+  @Override
+  public SiteadminActions createNewPage(String title, String templateName) {
+    toolbar.getCreatePageBtn().click();
+    toolbar.getCreatePageActions()
+        .selectTemplate(templateName)
+        .provideTitle(title)
+        .submit();
+
+    return this;
+  }
+
+  @Override
+  public SiteadminActions unpublishPage(String title) {
+    childPageWindow.selectPage(title);
+    toolbar.unpublishPage();
+    return this;
+  }
+
+  @Override
+  public SiteadminActions unpublishPageLater(String title, LocalDateTime localDateAndTime) {
+    childPageWindow.selectPage(title);
+    toolbar.unpublishPageLater(localDateAndTime);
+    return this;
+  }
+
+  @Override
+  public SiteadminActions deletePage(String title) {
+    childPageWindow.selectPage(title);
+    toolbar.deletePage();
+    return this;
+  }
+
+  @Override public SiteadminActions deleteSubPages() {
+    if (childPageWindow.hasSubPages()) {
+      childPageWindow.pressSelectAllPages();
+      toolbar.deletePage();
+    }
+    return this;
+  }
+
+  @Override
+  public boolean isPagePresent(String title) {
+    return childPageWindow.containsPage(title);
+  }
+
+  @Override
+  public SiteadminActions movePage(String title, String destinationPath) {
+    childPageWindow.selectPage(title);
+    toolbar.movePage(destinationPath);
+    return this;
+  }
+
+  @Override
+  public SiteadminActions open(String nodePath) {
+    driver.manage().addCookie(SiteadminLayout.LIST.getCookie());
+    driver.get(url + "/sites.html" + nodePath);
+    wait.withTimeout(Timeouts.SMALL).until(input -> isLoaded(), 2);
+    return this;
+  }
+
+  public void refresh() {
+    driver.navigate().refresh();
+  }
+
+  @Override
+  public SiteadminActions open() {
+    driver.get(url + "/sites.html");
+    driver.manage().addCookie(SiteadminLayout.LIST.getCookie());
+    driver.navigate().refresh();
+    return this;
+  }
+
+  @Override public boolean hasChildPages() {
+    return childPageWindow.hasSubPages();
+  }
+
+  public ChildPage getPageFromList(String title) {
+    return childPageWindow.getChildPage(title);
+  }
+
+  private void retryLoad() {
+    conditions.verify(new ExpectedCondition<Object>() {
+      @Nullable
+      @Override
+      public Object apply(WebDriver driver) {
+        driver.navigate().refresh();
```

Comment:
maybe it would be better to put refresh inside if?
``` 
if (!isLoadedCondition) {
      driver.navigate().refresh()
}
return isLoadedCondition()
```